pkg/server/commands/listusers/list_users_rpc_test.go

```diff
@@ -595,6 +595,59 @@ func TestListUsersCycles(t *testing.T) {
 			},
 			expectedUsers: []string{},
 		},
+		{
```

Comment:
**FYI:** I updated the upstream unit tests branch to include one more test cyclical test case.